hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/fees/calculation/consensus/txns/UpdateTopicResourceUsage.java

```diff
@@ -86,9 +80,8 @@ public FeeData usageGivenExplicit(
                         merkleTopic.hasAutoRenewAccountId(),
                         expiry,
                         txnBody.getConsensusUpdateTopic());
-            } catch (final InvalidKeyException illegal) {
-                log.warn("Usage estimation unexpectedly failed for {}!", txnBody, illegal);
-                throw new InvalidTxBodyException(illegal);
+            } catch (final InvalidKeyException ignore) {
+                // The key was validated before setting on MerkleTopic, so this should never happen
```

Comment:
I agree, this shouldn't happen, but in case it does, would it still be worth logging the warning so it doesn't fail silently?

Reply:
Makes sense. Done